libcloud/compute/drivers/gce.py

```diff
@@ -2243,7 +2251,14 @@ def ex_create_image(self, name, volume, description=None, family=None,
             image_data['rawDisk'] = {'source': volume, 'containerType': 'TAR'}
         else:
             raise ValueError('Source must be instance of StorageVolume or URI')
-
+        if guest_os_features:
+            image_data['guestOsFeatures'] = []
+            for feature in guest_os_features:
+                if feature in possble_features:
```

Comment:
Yup, thanks for the note. I missed the typo (and failing test) when I refactored some of the code).